root/static/scripts/edit/MB/edit.js

```diff
@@ -290,10 +290,10 @@ import {VIDEO_ATTRIBUTE_GID} from '../../common/constants';
 
 
     function editConstructor(type, callback) {
-        return function (args, orig) {
+        return function (args, ...rest) {
             args = _.extend({ edit_type: type }, args);
 
-            callback && callback.apply(null, arguments);
+            callback && callback.apply(null, [args].concat(rest));
```

Comment:
Was causing tests to fail in Node. Presumably https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Strict_mode/Transitioning_to_strict_mode#arguments_doesn't_alias_named_function_arguments

(We reassign `args` above, which in non-strict mode would be an alias for the corresponding `arguments` slot.)